scripts/test-pending-pacts.sh

```diff
@@ -12,4 +14,12 @@ cd ..
 
 cd provider
 sbt clean update test
+
+echo "Testing using the CLI with --includeWipPactsSince"
+sbt run &
```

Comment:
What does this run do?

Reply:
makes sure that the WIP field is passed correctly to the broker, which is all we can really test as the definition of what the broker considers WIP could change. The intitial test run just tests that pending pacts can be retrieved and handled correctly which, is deterministic